likecoin/admin/restful.php

```diff
@@ -144,10 +155,10 @@ function () {
 			);
 			register_rest_route(
 				'likecoin/v1',
-				'/publish-setting-page/matters-login',
+				'/publish-setting-page/logout-matters',
 				array(
-					'methods'             => 'POST',
-					'callback'            => 'likecoin_post_site_matters_login_data',
+					'methods'             => 'GET',
```

Comment:
Should use `POST` even if no body is needed, because `logout` modify the db state
In fact you can add `DELETE` using same url `login-matters` for more restful approach

Reply:
true! Will change that.